lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -200,10 +202,50 @@ getTransaction = isolate 43 $ do
             other -> fail $ "Invalid witness type: " ++ show other
 
 
+putTransaction :: (Tx, [TxWitness]) -> Put
+putTransaction (tx, witnesses) = do
+    putTokenTransfer tx
+    mapM_ putWitness witnesses
+
+putWitness :: TxWitness -> Put
+putWitness witness =
+    case witness of
+        PublicKeyWitness xPub (Hash sig) -> do
+            putWord8 1
+            putByteString xPub
+            putByteString sig
+        -- TODO: note that we are missing new address type witness:
```

Comment:
we still didn't cover acount and new address witness scheme

Reply:
Why though?

Reply:
@KtorZ 
@Anviking and I thoght that it would be ok to start with what we are confident with and extend other bits later

Reply:
Got you, though I am bit sad to see that we only merge this while the whole story was expected to be completed already 8 days ago.